pandas/core/generic.py

```diff
@@ -5074,7 +5074,9 @@ def pipe(self, func, *args, **kwargs):
         {see_also}
         Notes
         -----
-        `agg` is an alias for `aggregate`. Use the alias.
+        - `agg` is an alias for `aggregate`. Use the alias.
+        - Some NumPy functions such as ``np.mean``, ``np.nanmean``, ``np.median`` etc.
+            resolve to their corresponding internal cython function.
```

Comment:
Thanks @gurukiran07 - is the second line more intented than it needs to be?
```suggestion
        - Some NumPy functions such as ``np.mean``, ``np.nanmean``, ``np.median`` etc.
          resolve to their corresponding internal Cython function.
```

Second, I wonder if the significance of this is clear to a user - would it be clearer with an example in the "Examples" section? Perhaps show that you get the same output from `.agg(np.nanmedian)` and `.agg(np.median)`?

Reply:
Yes, agreed but it would be little distracting to add implementation details in the example IMO. Would be better if we elaborate a little more in the notes by adding *`.agg(np.median)`, `.agg(np.nanmedian)` will be handled in the same by the corresponding Cython function and similarly for `np.mean`, `np.nanmean` etc* What do you think?

Reply:
Sure - or just mention that, in general, pandas operations exclude NaNs

Reply:
@MarcoGorelli Perfect. Will edit it and push it. Thank you. And one more thing this applies to `transform` also right since `transform` is `agg` under the hood right? Should I add it there too?

Reply:
Should I mention *pandas operations like `"mean"`, `"median"` etc.* to make it more obvious. what do you think?

Reply:
How about (feel free to reword, of course)
```suggestion
        - Some NumPy functions resolve to their corresponding internal Cython function. As pandas operations
          generally exclude NaNs, this means that `.agg(np.nanmedian)`, `.agg(np.median)`, and `.agg('median')
          will return the same result.
```
?

I'll check about transform later (but if you've checked the source code already and have seen that that's the case, feel free to add this sentence to the transform docstring too)

Reply:
> How about  (feel free to reword, of course)

This is perfectly on point and short. I'll add these to transform too.

Link to transform  https://github.com/pandas-dev/pandas/blob/v1.0.5/pandas/core/generic.py#L10423
```python
@Appender(_shared_docs["transform"] % dict(axis="", **_shared_doc_kwargs))
    def transform(self, func, *args, **kwargs):
        result = self.agg(func, *args, **kwargs)
        if is_scalar(result) or len(result) != len(self):
            raise ValueError("transforms cannot produce aggregated results")

        return result
```